crates/lib-dialects/test/fixtures/dialects/tsql/declare_variable.yml

```diff
@@ -0,0 +1,11 @@
+file:
+- statement:
+  - keyword: DECLARE
+  - tsql_variable: '@MyVariable'
+  - data_type:
+    - data_type_identifier: INT
+  - comparison_operator:
+    - raw_comparison_operator: =
```

Comment:
nit: this isn't quite right. I don't even know though if we have an `assignment operator`

Reply:
Hm, I added this hope this helps: https://github.com/quarylabs/sqruff/pull/1664/commits/98c52d858c34171683fdebdde864064e2ce1c695

Reply:
But I will try another attempt using the `AssignmentOperator`.